pages/is-plotly-free.md

```diff
@@ -0,0 +1,23 @@
+## Is Plotly for Python Free?
+
+&nbsp;  &nbsp; **Yes.** &nbsp; Plotly for Python is free and open-source software, [licensed under the **MIT license**](https://github.com/plotly/plotly.py/blob/master/LICENSE.txt). It costs nothing to [install and use](/python/getting-started). You can view the source, report issues or contribute using [our Github repository](https://github.com/plotly/plotly.py).
```

Comment:
why the forced indent with `&nbsp;` ?

Reply:
No reason. I just kept it as is from https://github.com/plotly/graphing-library-docs/blob/master/_posts/python/2019-07-03-is-plotly-free.md
But yeah, I don't think it's necessary. Will remove it